src/Pages/Cart/CartComponents/AmountCard/AmountCard.jsx

```diff
@@ -0,0 +1,40 @@
+import { useEffect, useState } from "react";
+import { useCartContext } from "../../../../context/Cart.context";
+import "./AmountCard.css";
+
+export const AmountCard = () => {
+  const { cartProducts } = useCartContext();
+
+  const [total, setTotal] = useState();
+
+  useEffect(() => {
+    setTotal(
+      cartProducts.reduce((acc, curr) => acc + Number(curr.price) * curr.qty, 0)
+    );
+  }, [cartProducts]);
+
+  return (
+    <div className="card-container card-shadow">
+      <h2 className="card-subtitle bold">Price Detail</h2>
+      <div className="filter-divider-line"></div>
+      <h2 className="card-subtitle">
```

Comment:
Here, `<h2>` are repeating so you can use an array here to map.

You can make an array of objects.

Reply:
sure will think on this